blockdevice/blockdevice.go

```diff
@@ -21,9 +21,12 @@ package blockdevice
 //
 // 1. Example blockdevice struct for a partition /dev/sda1
 // 		{
-// 				UUID:blockdevice-4c25d69f9adc868f61e3d891cf3a5613
+// 				Identifier:{
```

Comment:
this is a breaking change. hoping other dependant components are aware if this.

Reply:
Its in the internal struct, not in the kubernetes API resource.